modules/installation-initializing-manual.adoc

```diff
@@ -159,6 +168,19 @@ platform:
 <4> Specify the name of your Azure Stack Hub region.
 endif::ash[]
 
+ifdef::ash-default,ash-custom[]
++
+Make the following modifications:
+
+.. Specify the required installation parameters.
```

Comment:
If you're reading this straight through, it is confusing because I think it refers to the next section, Maybe it should say "Specify the required installation parameters from the next section"

Reply:
This is, unfortunately, a drawback of how the documentation is written in reusable modules. While specifying "next section" would work in this particular topic, it would likely cause confusion in another section of the help that resuses this content, but does not include the installation configuration parameters.